packages/shield-controller/src/backend.ts

```diff
@@ -0,0 +1,117 @@
+import type { TransactionMeta } from '@metamask/transaction-controller';
+
+import type { CoverageResult, CoverageStatus, ShieldBackend } from './types';
+
+export const BASE_URL = 'https://rule-engine.metamask.io';
```

Comment:
endpoint should be injected in controller config

Reply:
TranasctionController is doing it like this for Sentinel URL, but will change it. I also think that it's good to have it configurable.